lib/src/CalendarSkeleton.tsx

```diff
@@ -9,7 +9,7 @@ import { useStyles as useCalendarStyles } from './views/Calendar/Calendar';
 export interface CalendarSkeletonProps extends React.HTMLProps<HTMLDivElement> {}
 
 const muiComponentConfig = {
-  name: 'MuiPickersCalendarSkeleton',
+  name: 'MuiPickerCalendarSkeleton',
```

Comment:
If you feel that the component is specific enough to no conflict in the future of the whole MUI codebase:

```suggestion
  name: 'MuiCalendarSkeleton',
```

Otherwise

```suggestion
  name: 'MuiPickersCalendarSkeleton',
```